### ISSUE TITLE
Shell Sort [Java]

### ISSUE THREAD

**Issue by jsroyal** [iiitv/algos#391]
#### This issue is about [Algo/DS Name](link to resource for the Algo/DS).

- [X] Issue Title is in form `Algo/DS Name [Language]`.
- [X] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
- [X] This Algo/DS is worth adding to this repository.
- [X] Assign this issue to me (I want to work on this).
	- [X] I ensure that I am not already assigned to 2 or more issues.
- [x] I will send my PR only after I'm assigned to this issue by any maintainer.
- [X] I understand that I will be unassigned from this issue if I fail to send a PR for this within a week.

**Comment by jsroyal** [comment]
>[] I will send my PR only after I'm assigned to this issue by any maintainer.
does it mean to wait to assign?

**Comment by Monal5031** [comment]
@jsroyal Why did you close this one?

**Comment by jsroyal** [comment]
@Monal5031 It is done. And merged into master.
